src/com/dotmarketing/servlets/ShortyServlet.java

```diff
@@ -71,8 +71,19 @@ private void _serve(final HttpServletRequest request, final HttpServletResponse
       return;
     }
 
-    final String uri = request.getRequestURI().toLowerCase();
+    String uri = request.getRequestURI();
+    
+    
+    StringTokenizer tokens = new StringTokenizer(uri, "/");
+    if (tokens.countTokens() < 2) {
+      response.sendError(404);
+      return;
+    }
 
+    tokens.nextToken();
+    String id = tokens.nextToken();
+    String fieldName = tokens.hasMoreTokens() ? tokens.nextToken() : NOT_FOUND;
```

Comment:
This fixes a bug whereby Shorty can't serve a non-defualt field that has a camelcase velocity variable